### DISCUSSION TITLE
Subscription Count

### DISCUSSION THREAD

**Discussion by Jim-Webfox** [soketi/pws#800] [Ideas]
Would anyone be keen on having a feature replicating the pusher feature for counting live users at scale? (https://blog.pusher.com/counting-live-users-at-scale-with-subscription-count-events/) 

I have it half implemented on one of my forks and am about to move onto rate limiting when over 100 (or user configurable) connections. But want to know if it's something that people would be interested in?

I plan to keep it as similar as I can to how it works with pusher so it can still be a drop in replacement.

Cheers

**Comment by nickchomey** [comment]
That would be very cool

**Comment by Jim-Webfox** [comment]
https://github.com/soketi/soketi/pull/819

**Comment by tinusg** [comment]
I would love this. I considered adding webhooks to my app to keep track of the connected clients, but that seems like a lot of overhead for something as basic as keeping count.
